e2e/suites/actions-available/files-folders/test-data.ts

```diff
@@ -382,13 +382,12 @@ const searchDocxContextMenu = [
   'Edit Aspects',
   'Permissions'
 ];
-const searchToolbarPrimary = ['Toggle search filter', 'Share', 'Download', 'View', 'View Details', 'More Actions'];
-const searchSharedToolbarPrimary = ['Toggle search filter', 'Shared Link Settings', 'Download', 'View', 'View Details', 'More Actions'];
+const searchToolbarPrimary = ['Toggle search filter', 'Share', 'Download', 'View Details', 'More Actions'];
```

Comment:
why is the View action being removed? 
I don't see this in the acceptance criteria of the story.

Reply:
You won't see anymore view on the main toolbar but you will go through the viewer into the rightside . The main toolbar and the filter will be refactored with coming up stories.

Reply:
so you're moving the View action that was accessible by a single click, in the Info Drawer, making the user do 2 clicks. 
Is this ok? 
The acceptance criteria say "When viewing search results users can initiate a preview of a specific document with 1-click."

Reply:
This is what this whole feature is about, long story short instead of having it on the top bar and opening in overlay it will open up with 1-click from the info drawer. It will change the file while navigating through them also. I can double check the feature and maybe clarify some point

Reply:
Just to summarize what happens : I'll re enable the view action even though the 1-click view can be performed through the name click, but even if it is redundant for now, there are other stories to refactor completely that part.